client-ts/package.json

```diff
@@ -1,47 +1,24 @@
 {
-  "name": "@aspnet/signalr-client",
-  "version": "1.0.0-preview1-t000",
-  "description": "ASP.NET Core SignalR",
-  "main": "dist/src/index.js",
-  "typings": "dist/src/index.d.ts",
-  "directories": {
-    "test": "test"
-  },
+  "name": "client-ts",
+  "version": "0.0.0",
+  "description": "Common dev-time dependencies for SignalR NPM packages. NOT FOR PUBLISHING",
+  "main": "index.js",
   "scripts": {
-    "gulp": "gulp",
-    "pretest": "tsc -p Microsoft.AspNetCore.SignalR.Client.TS.Tests",
-    "test": "jasmine JASMINE_CONFIG_PATH=Microsoft.AspNetCore.SignalR.Client.TS.Tests/jasmine.json"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/aspnet/SignalR.git"
+    "build": "cd ./signalr && npm run build && cd ../signalr-protocol-msgpack && npm run build && cd ../FunctionalTests && npm run build",
+    "test": "cd ./signalr && npm run test && cd ../signalr-protocol-msgpack && npm run test"
```

Comment:
My kingdom for JSON comments.

These scripts are just for convenience. You can run `npm run build` or `npm test` in the `client-ts` folder and it will build/test all the subfolders.